lib/generators/reactionview/install_generator.rb

```diff
@@ -21,7 +21,7 @@ def create_initializer
             # config.intercept_erb = true
 
             # Enable debug mode in development (adds debug attributes to HTML)
-            config.debug_mode = Rails.env.development?
+            config.debug_mode = Rails.env.development? && ENV.fetch("REACTIONVIEW_DEBUG_MODE", "true") == "true"
```

Comment:
```suggestion
            config.debug_mode = Rails.env.development? && !ENV["REACTIONVIEW_DISABLE_DEBUG_MODE"]
```

@t27duck Thanks for the feedback! You make a great point about the simplicity of checking for the presence of the environment variable rather than its specific value.

You're absolutely right that this is cleaner - any value ("1", "yes", "TRUE", etc.) would disable debug mode, which is more intuitive and follows common Unix conventions.

My initial implementation was more verbose because I like environment variables that explicitly say what they're doing (like `REACTIONVIEW_DEBUG_MODE=false`), but at the end of the day, this is really just a matter of taste and your approach is definitely more concise.

The important thing is that we're solving the core problem: allowing individual developers in a team to control debug mode through their personal environment setup without changing code that affects everyone else. Whether someone prefers `REACTIONVIEW_DISABLE_DEBUG_MODE=1` or `REACTIONVIEW_DEBUG_MODE=false` is secondary to having that flexibility.

I'm open to updating the implementation either way 🙂